core/src/main/java/jenkins/util/ProgressiveRendering.java

```diff
@@ -203,6 +203,13 @@ private static void setCurrentRequest(RequestImpl request) {
         }
     }
 
+    /**
+     * @return whether the computation has finished.
+     */
+    public boolean isFinished() {
```

Comment:
If this is test-only, it should be restricted

Reply:
fixed ;)